### ISSUE TITLE
Feature Request: Please add MySQL database

### ISSUE THREAD

**Issue by abdusamadzoidov17** [Automattic/studio#584]
### What

Please add MySQL database

### Why

Please add MySQL database

**Comment by wojtekn** [comment]
HI @abdusamadzoidov17 . Thanks for adding the feature request.

Does SQLite database prevent you from using Studio? Did you try importing the MySQL database to Studio?

**Comment by vincent90-wp** [comment]
I would also like to encourage the adoption of MySQL, as many plugins are specifically designed for it. Additionally, importing a MySQL database can cause issues, at least when using All-in-One Migration. Given that hosting services optimized for WordPress are already configured for MySQL and MariaDB, using MySQL would make everything much easier and more intuitive.

**Comment by gettonet** [comment]
I started a fresh site in Studio, and used my normal flow for setting it up. When it came to deploying the website to a live server, it took me hours to understand that all issues were due to SQLite.
Try installing WPML or Duplicator, it gets messed up immediately.
Also, it is not possible to view/edit the database from Studio, which to me is a huge minus comparing to Local, for example, and a feature which must be available for any developer.


UPDATE:

Ok, I found outthat there is a way to access the database: https://developer.wordpress.com/docs/developer-tools/studio/frequently-asked-questions/#13-how-can-i-manage-the-studio-site-s-sqlite-database-

Still, looks like that there are bunch of plugins which are incompatible with SQLite.

**Comment by georgestephanis** [comment]
Noting that there are ways detailed in 👆 to support using MySQL locally with Studio --

https://developer.wordpress.com/docs/developer-tools/studio/frequently-asked-questions/#use-studio-with-mysql-server

I think it's good for it to default to SQLite, with the ability for those that need it to use MySQL instead, personally.

**Comment by nicdwilson** [comment]
> Noting that there are ways detailed in 👆 to support using MySQL locally with Studio

I agree, in the context of Playground SQLite makes sense, but the ability to fall back to MySQL is limited too. WooCommerce will not activate on a MySQL Studio site.

**Comment by faizan-official** [comment]
Even the Elementor Page builder doesn't work well with SQLite database, there should be separate tab or setting to configure database where we can enter details like this attached VS Code plugin, not exactly all of these but something which can help with MySQL and MariaDB at least and default to SQLite.

<img width="1171" height="784" alt="Image" src="https://github.com/user-attachments/assets/39cb6309-4fe2-462f-899e-a238d93a0455" />

**Comment by TheJoelGray** [comment]
I'm in agreement with the ability to change to MySQL up front. I've had issues on about 50% of the sites I've tried to import or sync due to various plugins that are incompatibe with SQLite. It ususally manifests as a problem during the import.

**Comment by Shankowsky** [comment]
I just stopped using Studio and used a linux approach with a docker. It wasn't worth the headache to use SQLite. Nearly all hosts use MySQL, a lot of plugins us MySQL. So why go against the whole internet.
